corehq/apps/mocha/templates/mocha/base.html

```diff
@@ -41,6 +41,10 @@
       {% block fixtures %}{% endblock %}
       </div>
 
+      <div class="initial-page-data" class="hide">
```

Comment:
I'd think you could leave this div div out of base.html altogether, since `register` is manipulating the js object directly.

Reply:
good point, removed